src/storage/pooled_storage_manager.h

```diff
@@ -57,6 +57,7 @@ class GPUPooledStorageManager final : public StorageManager {
   GPUPooledStorageManager() {
     reserve_ = dmlc::GetEnv("MXNET_GPU_MEM_POOL_RESERVE", 5);
     page_size_ = dmlc::GetEnv("MXNET_GPU_MEM_POOL_PAGE_SIZE", 4096);
+    large_alloc_round_size_ = dmlc::GetEnv("MXNET_GPU_MEM_LARGE_ALLOC_ROUND_SIZE", 2 * 1024 * 1024);
```

Comment:
I developed another rounding memory manager that uses a combination of exponential rounding size and linear rounding size and has shown good results. If adding rounding to GPU is desired by default, I think we can combine them into a single memory manager that offer these knobs.

let's address the memory pool change in a separate PR.